Sources/SwiftDocC/Infrastructure/Diagnostics/TerminalHelper.swift

```diff
@@ -0,0 +1,17 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2023 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See https://swift.org/LICENSE.txt for license information
+ See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import Darwin.C
```

Comment:
I missed this until the CI built this for Linux. 

This only works on Darwin platforms (macOS, iOS, etc.). I think this import would need to look something like this:

```suggestion
#if canImport(Darwin)
import Darwin.C
#if canImport(Glibc)
import Glibc
#elseif canImport(Musl)
import Musl
#elseif os(Windows)
import CRT
#endif
```

The above is based on [this Forum post](https://forums.swift.org/t/pitch-the-cstdlib-module/51373) and [these imports](https://github.com/apple/swift-tools-support-core/blob/580248410c553f08fc9f01d0246453f89e6192b1/Sources/TSCLibc/libc.swift#L11C1-L20C7).